iOS/MasterFeed/MasterFeedViewController.swift

```diff
@@ -564,6 +564,8 @@ class MasterFeedViewController: UITableViewController, UndoableCommandRunner, Ma
 			return
 		}
 		
+		if tableView.window == nil { return }
```

Comment:
We still should call the completion — and we probably do want to call `reloadData` too, which I think is smart enough not to do layout when it’s not actually warranted. (I think.)

Reply:
From testing, we can call the completion safely, but the `reloadData` opens up another avenue to hit this bug — it appears to trigger an unreadCountDidChange notification, which trickles down to a layout of the cell

<img width="516" alt="Screenshot 2023-03-13 at 13 44 49" src="https://user-images.githubusercontent.com/7046652/224618150-43f57865-3f04-48d5-bf67-4a1805b0b22c.png">